homeassistant/components/acmeda/helpers.py

```diff
@@ -0,0 +1,37 @@
+"""Helper functions for Acmeda Pulse."""
+from homeassistant.helpers.device_registry import async_get_registry as get_dev_reg
+
+from .const import DOMAIN, LOGGER
+
+
+async def add_entities(hass, entity_class, config_entry, current, async_add_entities):
```

Comment:
This can be a callback since we don't await inside.
```suggestion
@callback
def async_add_acmeda_entities(hass, entity_class, config_entry, current, async_add_entities):
```

Reply:
Changed to callback, similarly changed calling functions in sensor and cover.